rm -rf build
git reset --hard && git checkout . -f && git submodule update --checkout -f 
patch -Np1 -i "${mypatd:-"../"}/psmoveapi.patch"
mkdir build && cd build && cmake -DCMAKE_BUILD_TYPE=Release -DCMAKE_INSTALL_PREFIX=/usr ..
diff a/examples/c/distance_calibration.cpp b/examples/c/distance_calibration.cpp
index c65e813..eb853f6 100644
--- a/examples/c/distance_calibration.cpp	1969-12-31 17:00:00.000000000 -0700
+++ b/examples/c/distance_calibration.cpp	1969-12-31 17:00:00.000000000 -0700
@@ -35,6 +35,7 @@
 #include "opencv2/opencv_modules.hpp"
 #include "opencv2/core/core_c.h"
 #include "opencv2/highgui/highgui_c.h"
+#include "opencv2/imgcodecs/imgcodecs_c.h"
 
 #include "psmove.h"
 #include "psmove_tracker.h"
diff a/src/tracker/camera_control.h b/src/tracker/camera_control.h
index aa2aaee..144f763 100644
--- a/src/tracker/camera_control.h	1969-12-31 17:00:00.000000000 -0700
+++ b/src/tracker/camera_control.h	1969-12-31 17:00:00.000000000 -0700
@@ -28,6 +28,7 @@
  **/
 
 #include "opencv2/core/core_c.h"
+#include "opencv2/videoio/videoio_c.h"
 
 #ifdef __cplusplus
 extern "C" {
